designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ExpressionSuggesterResources.scala

```diff
@@ -0,0 +1,83 @@
+package pl.touk.nussknacker.ui.api
+
+import akka.http.scaladsl.server.{Directives, Route}
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import io.circe.generic.JsonCodec
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+
+import scala.collection.Map
+import scala.concurrent.ExecutionContext
+
+class ExpressionSuggesterResources(expressionSuggester: ExpressionSuggester)(implicit ec: ExecutionContext) extends Directives with FailFastCirceSupport with RouteWithUser {
+  def securedRoute(implicit user: LoggedUser): Route =
+    path("expressionSuggestions") {
```

Comment:
I think that we should move this resource next to `NodesResources`: POST parameters" / Segment / validate.
We should do that because of the reasons below:
- We have to at least know the processingType (see `DefinitionResources`: "processDefinitionData" / Segment)
- But our typer currently provides also value types so `NodesResources` will be a better place (it has full knowledge of scenario's `MetaData`. Using processName we can find processingType as well